cclib/parser/molcasparser.py

```diff
@@ -98,7 +110,7 @@ def extract(self, inputfile, line):
             self.atomcoords.append(atomcoords)
 
             if self.atomnos == []:
-                self.atomnos = [utils.PeriodicTable().number[atomelement] for atomelement in atomelements]
+                self.atomnos = [utils.PeriodicTable().number[ae.title()] for ae in atomelements]
```

Comment:
The Logfile also has a PeriodicTable stored:
```suggestion
                self.atomnos = [self.table.number[ae.title()] for ae in atomelements]
```